backend/server/Cargo.toml

```diff
@@ -35,22 +51,23 @@ graphql = { path = "../graphql" }
 graphql_v1 = { path = "../graphql_v1" }
 graphql_core = { path = "../graphql/core" }
 graphql_types = { path = "../graphql/types" }
-dgraph = {path = "../dgraph"}
+dgraph = { path = "../dgraph" }
 repository = { path = "../repository" }
 service = { path = "../service" }
 util = { path = "../util" }
 
 actix-cors = "0.6.1"
-actix-web = { version= "4.0.1" } 
-actix-http ="3.3.1"
+actix-web = { version = "4.0.1" }
+actix-http = "3.3.1"
 config = "0.13"
 log = "0.4.14"
 serde = "1.0.137"
-tokio = { version = "1.29", features = ["macros" ] }
+tokio = { version = "1.29", features = ["macros"] }
 rust-embed = "6.4.2"
 mime_guess = "2.0.4"
 futures-util = "0.3"
 simple-log = { version = "1.6" }
+listenfd = "1.0.1"
```

Comment:
ah my formatter had fun in this file, it's just this line adding `listenfd` for the TCP listener that actually changed here